ls1intum/Artemis
# `Quiz exercise`: Show the last grades in practice mode

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


On the quiz exercises scores page in the instructor perspective, the grades is not shown for the attempts in practice mode. Fixed the open issue #7150.

### Description

I changed the html file for the exercise scores webpage. In the cell of the datatable shown in this page, the option of shownUngradedResults was set to false, and now is set to true. So grades of non-graded attempts in practice mode could also be shown in the table.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course
- 1 Quiz Exercise

Instructor:
1. Create a Quiz Exercise
Student
2. Solve the quiz (wait until the quiz ends)
3. Verify your grade
Instructor: 
4. Verify the student's grade
5. Release the quiz for practice
Student:
7.  Practice the quiz
8. Check the new obtained grade 
9. Verify in statistics that your previous grade didn't change
Instructor:
10. In score of the quiz exercise, check the new grade obtained by the student
11. In the score of the course, check that the previous score didn't change. 

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots



The scores button could be found here in Course Management -> Quiz Exercises:

<img width="1392" alt="screenshot 1" src="https://github.com/ls1intum/Artemis/assets/138691856/a3d90b29-2717-4111-9da2-30fbd7ddb817">

Before, no graded result is shown for the second attempt in practice mode:

<img width="901" alt="screenshot 2" src="https://github.com/ls1intum/Artemis/assets/138691856/8a6de379-8a1e-41e3-a191-66c6a257e21e">


After, result is shown there:

<img width="906" alt="screenshot 3" src="https://github.com/ls1intum/Artemis/assets/138691856/bd204654-aaad-4f0b-bace-cf1ca516a2bc">
